Changes.md

```diff
@@ -1,5 +1,25 @@
 # Revision history for the Workflow Perl Distribution
 
+## DEPRECATION NOTICE
+
+- With release 2.00 Workflow::Persister::SPOPS will no longer be included in the distribution, it will possibly be made available as a separate distribution, but with decreased maintenance efforts. [SPOPS](https://metacpan.org/pod/SPOPS) does no longer seem to be actively supported and [issues with Perls versions from 5.11.1 and onwards](http://matrix.cpantesters.org/?dist=SPOPS+0.87) underlines this fact.
```

Comment:
`SPOPS+0.87) underlines this fact.` -> `SPOPS+0.87) underline this fact.` ?

Reply:
Fixed